### DISCUSSION TITLE
How do you delete an edge?

### DISCUSSION THREAD

**Discussion by jtwigg** [reaviz/reaflow#157] [Q&A]
I cannot find an example that allows me to delete an edge.
Nor an example that allows me to even add an edge. 

Can someone highlight how to do this?

**Comment by Faizan-e-Muhammad97** [comment]
There is an example for adding edges between nodes. 
[Linking Nodes](https://reaflow.dev/?path=/story/demos-nodes--linking-nodes)

I couldn't find any proper example for deleting edges. However I was able to code it.

```
edge={
          <Edge
            onClick={(event, node) => {
              onClick && onClick(event, node);
            }}
            onRemove={(event, edge) => {
              const result = filter(edges, (e) => {
                if (e.id === edge.id) return false;
                return true;
              })
              setEdges(result);
              clearSelections();
            }}
          />
        }
```

It is inspired from the selections demo code.
[Default Selections](https://reaflow.dev/?path=/story/demos-selections--defaults)
